docs/guides/installation-into-an-existing-rails-app.md

```diff
@@ -7,48 +7,45 @@
 1. Add the following to your Gemfile and `bundle install`. We recommend fixing the version of React on Rails, as you will need to keep the exact version in sync with the version in your `package.json` file.
 
    ```ruby
-   gem "react_on_rails", "12.0.0" # Update to the current version
-   gem "webpacker", "~> 5"
+   gem "react_on_rails", "13.1.0" # Use the latest and the exact version
+   gem "shakapacker", "6.5.4"
    ```
 
-1. Add the webpacker and react_on_rails gems
-_Use the latest version for react_on_rails._
+   Or use `bundle add`:
 
-```
-bundle add webpacker                 
-bundle add react_on_rails --version=12.0.4 --strict
-```
+   ```bash
+   $ bundle add react_on_rails --version=13.1.0 --strict
+   $ bundle add shakapacker
```

Comment:
@ahangarha what is the `$` for? your personal prompt?

Reply:
Thanks for bringing this up. This was one of my concerns and since it was minor, I didn't bring it up. In our docs, sometimes we have this `$` sign (which represents the shell of a non-root user). To me, not only this is not necessary (the user should know which code is to run in the shell and which one to is to add to a file), but it is annoying especially when one wants to copy/paste the code block.

It should be enough to have syntax highlighting.

Now I am confident to remove `$` from all such cases.

Reply:
@justin808 Done